Makefile

```diff
@@ -13,6 +13,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+GO=GOOS=linux GO111MODULE=off go
```

Comment:
Why GOOS=linux?

Reply:
> Why GOOS=linux?
This is not added part of this change. It was existing before, I just moved from that place.

Reply:
We weren't consistent then. I was looking at the first changed line and that only had:
```
build: $(CMDS) $(TEST_CMDS)
	go test -run none ./pkg/... ./test/e2e
```